shell/platform/android/platform_view_android_jni.cc

```diff
@@ -174,18 +174,26 @@ std::unique_ptr<IsolateConfiguration> CreateIsolateConfiguration(
   const auto configuration_from_blob =
       [&asset_manager](const std::string& snapshot_name)
       -> std::unique_ptr<IsolateConfiguration> {
-    std::unique_ptr<fml::Mapping> blob =
-        asset_manager.GetAsMapping(snapshot_name);
+    auto blob = asset_manager.GetAsMapping(snapshot_name);
+    auto delta = asset_manager.GetAsMapping("kernel_delta.bin");
+    if (blob && delta) {
+      std::vector<std::unique_ptr<fml::Mapping>> kernels;
+      kernels.emplace_back(std::move(blob));
+      kernels.emplace_back(std::move(delta));
+      return IsolateConfiguration::CreateForKernelList(std::move(kernels));
+    }
     if (blob) {
       return IsolateConfiguration::CreateForSnapshot(std::move(blob));
     }
+    if (delta) {
```

Comment:
When would have only "delta"?

Reply:
When running corejit in release or profile mode. In that case, both framework and host app are part of snapshot (so we don't distribute kernel_blob.bin). Before this PR, both kernel_blob.bin and kernel_delta.bin were called kernel_blob.bin, while in reality they're different and can be used together (like in debug mode above).

Reply:
Okay, I see.